src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -278,13 +278,6 @@ export const buildPipelineVisFunction: BuildPipelineVisFunction = {
     const paramsArray = [paramsJson, uiStateJson].filter((param) => Boolean(param));
     return `tsvb ${paramsArray.join(' ')}`;
   },
-  table: (params, schemas) => {
-    const visConfig = {
-      ...params,
-      ...buildVisConfig.table(schemas, params),
-    };
-    return `opensearch_dashboards_table ${prepareJson('visConfig', visConfig)}`;
-  },
```

Comment:
Just want to check my understanding here - this is safe to remove because in the re-written table vis, we define a `to_ast` function so that we'll never hit the `else` condition in https://github.com/opensearch-project/OpenSearch-Dashboards/blob/cf4a4b2e80f8c3417c945db5526d9ebf62a5d5e0/src/plugins/visualizations/public/legacy/build_pipeline.ts#L420-L423

Is that correct?

Reply:
yep!